src/plugins/intel_cpu/tests/functional/test_utils/x64/filter_cpu_params.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/filter_cpu_params.hpp"
+#include "ie_ngraph_utils.hpp"
```

Comment:
is it possible to not use `ie_ngraph_utils.hpp`?

Reply:
Done